File: BAEKJOON/all_problems/31K/31656.js
const fs = require('fs');
const inputData = fs.readFileSync(0, 'utf-8');
// console.log(inputData);

const n = inputData.length;
let ans = inputData[0];
let pre = inputData[0];

for (let i = 1; i < n; ++i) {
  if (pre == inputData[i]) continue;
  pre = inputData[i];
  ans += inputData[i];
}

console.log(ans);

File: BAEKJOON/all_problems/31K/31656.cpp
#include <bits/stdc++.h>
using namespace std;

int main(){
    string s; getline(cin, s);
    int n = s.length();

    string ans = "";
    ans += s[0];
    char pre = s[0];

    for(int i=1; i<n; ++i){
        if(pre == s[i]) continue;
        pre = s[i];
        ans += s[i];
    }

    cout << ans << '\n';
    return 0;
}